.github/workflows/release-embedding-sdk.yml

```diff
@@ -3,7 +3,6 @@ name: Release Metabase Embedding SDK for React
 on:
   workflow_dispatch:
     inputs:
-      # TODO: Add a version validation, so the workflow won't fail when publishing to npm
```

Comment:
This comment isn't valid anymore since we have now automated the version number when publishing the SDK npm package.